registry/templates/401.html

```diff
@@ -0,0 +1,9 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+  <h1>Jste odhlášeni</h1>
```

Comment:
Uvažuji, zda není lepší tam napsat _Nejste přihlášeni_, ale to je asi věc osobní preference. :smile:

Reply:
Opraveno.